### PR TITLE
Add links of database talks, building database from strach, and Stanford course.

### PR DESCRIPTION
**Pull request by laixintao**
Hi Pingcap, nice repo!

I myself also collected some links here: https://www.kawabangga.com/db , I will submit more later :)

### CODE REVIEW

**Comment by laixintao** [comment]
Regenerated Toc using vim plugin, add `talks` section.

Ready to merge, PTAL @zz-jason

**Comment by laixintao** [comment]
ping @zz-jason

File: README.md [hunk 1]
```diff
@@ -83,6 +92,7 @@ Blogs:
     - [数据库内核杂谈（七）：数据库优化器（上）](https://www.infoq.cn/article/GhhQlV10HWLFQjTTxRtA)
     - [数据库内核杂谈（八）：数据库优化器（下）](https://www.infoq.cn/article/JCJyMrGDQHl8osMFQ7ZR)
 - [SQL优化器原理 - 查询优化器综述](https://yq.aliyun.com/articles/610128), thanks to [勿烦](https://yq.aliyun.com/users/kyni3qcv656rk?spm=a2c4e.11153940.0.0.6adc1a8etfb0vx)
+- [阿里巴巴数据库内核月报](http://mysql.taobao.org/monthly/)
```
Comment (zz-jason) [inline README.md]: this is a series of blogs, it's better to only pick the blogs related to sql grammar and relational algebra to this section.

File: README.md [hunk 2]
```diff
@@ -306,3 +317,8 @@ Blogs:
 - [Building a Large-scale Distributed Storage System Based on Raft](https://www.cncf.io/blog/2019/11/04/building-a-large-scale-distributed-storage-system-based-on-raft/), by Ed Huang
 
 ## Benchmark & Testing
+
+## Build A Database from Scratch
+- [Let's Build a Simple Database](https://cstack.github.io/db_tutorial/)
+- [builddatabase](https://github.com/ngaut/builddatabase)
+- [Writing a sqlite clone from scratch in C](https://github.com/cstack/db_tutorial)
```
Comment (zz-jason) [inline README.md]: this is the same with the first one "Let's Build a Simple Database"

File: README.md [hunk 3]
```diff
@@ -306,3 +317,8 @@ Blogs:
 - [Building a Large-scale Distributed Storage System Based on Raft](https://www.cncf.io/blog/2019/11/04/building-a-large-scale-distributed-storage-system-based-on-raft/), by Ed Huang
 
 ## Benchmark & Testing
+
+## Build A Database from Scratch
+- [Let's Build a Simple Database](https://cstack.github.io/db_tutorial/)
+- [builddatabase](https://github.com/ngaut/builddatabase)
```
Comment (zz-jason) [inline README.md]: most content of this repo is empty, we can add this material when it's finished.

File: README.md [hunk 4]
```diff
@@ -298,6 +308,7 @@ Courses:
 ## Consensus
 
 - University of Cambridge [Distributed consensus revised](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-935.pdf), a great paper about Consenssus especially Paxos and Paxos-Related algorithms, by Heidi Howard
+- [Ark: A Real-World Consensus Implementation By Zardosht Kashef](https://arxiv.org/pdf/1407.4765.pdf)
```
Comment (zz-jason) [inline README.md]: for papers, it's better to be formatted like this:
```
Papers:

- 2014, [Ark: A Real-World Consensus Implementation](https://arxiv.org/pdf/1407.4765.pdf), CoRR
```

BTW, the first one is a technical report, it's better to separate them into different sections.

File: README.md (hunk 1 above)
Comment (laixintao) [reply README.md]: The series is valuable from my point.

Any section I can move the index link there?

File: README.md (hunk 2 above)
Comment (laixintao) [reply README.md]: Yes sorry.

File: README.md (hunk 1 above)
Comment (laixintao) [reply README.md]: This blog is also great: https://github.com/digoal/blog

I hope there is a section to put blog series like this.

File: README.md (hunk 4 above)
Comment (laixintao) [reply README.md]: Agreed

File: README.md (hunk 1 above)
Comment (laixintao) [reply README.md]: I deleted them for now so you can merge this PR. Let me know if you have places to accept blog series, I will submit them in another PR. :)

File: README.md (hunk 4 above)
Comment (zz-jason) [reply README.md]: I mean "Distributed consensus revised" is a technical report, while "Ark: A Real-World Consensus Implementation" is a research paper. For this section, it's recommended to be formatted like this:

```

Technical Report:
- University of Cambridge [Distributed consensus revised](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-935.pdf), a great paper about Consenssus especially Paxos and Paxos-Related algorithms, by Heidi Howard

Paper:
 
- 2014, [Ark: A Real-World Consensus Implementation](https://arxiv.org/pdf/1407.4765.pdf), CoRR
```

Sorry for the misleading 😂

Comment (laixintao) [reply README.md]: OK now I get you. updated.

**Comment by laixintao** [comment]
Hi @zz-jason , will you take a look again?

**Comment by laixintao** [comment]
ping @zz-jason

**Comment by zz-jason** [comment]
> ping @zz-jason

Sorry for the late reply.

**Pull request merged**